package.json

```diff
@@ -6,6 +6,7 @@
     "docs-dev": "vitepress dev devdoc",
     "docs-build": "vitepress build devdoc",
     "docs-preview": "vitepress preview devdoc",
+    "convert": "pandoc -w markdown -o ./docs/DOCUMENTATION.md ./docs/DOCUMENTATION.rst",
```

Comment:
still needed?

Reply:
Yes - so I don't have to work out how to do it next time!